### ISSUE TITLE
Not working with Mac M1

### ISSUE THREAD

**Issue by VoltixIGS** [Thizzer/jtouchbar#60]
I used the swing version and every time I try to comple and execute I get this error 

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/26350600/143939141-7a506388-b1b7-40c2-8347-e52edcff060a.png">

**Comment by derreisende77** [comment]
Currently there is no support for an Apple Silicon built native library in jtouchbar.

**Comment by VoltixIGS** [comment]
You know if there will be?

**Comment by derreisende77** [comment]
No, I don´t know. I am not involved with this library. Maybe @ttveldhuis can answer if there will be support in the future.
